chapters/mvc-basics/use-dependency-injection.md

```diff
@@ -79,12 +79,14 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
-Add a new line anywhere inside the `ConfigureServices` method that tells ASP.NET Core to use the `FakeTodoItemService` for the `ITodoItemService` interface:
+The job of the `ConfigureServices` method is adding things to the **service container**, or the collection of services that ASP.NET Core knows about. The `services.AddMvc` line adds the services that the internal ASP.NET Core systems need (as an experiment, try commenting out this line). Any other services you want to use in your application must be added to the service container here in `ConfigureServices`.
+
+Add a new line anywhere inside the method that tells ASP.NET Core to use the `FakeTodoItemService` whenever the `ITodoItemService` interface is requested:
```

Comment:
would change "a new line" to "the following line", new line sounds like the character. xD